app/code/core/Mage/Catalog/Model/Product/Attribute/Backend/Startdate.php

```diff
@@ -39,9 +39,6 @@ protected function _getValueForSave($object)
     {
         $attributeName  = $this->getAttribute()->getName();
         $startDate      = $object->getData($attributeName);
-        if ($startDate === false) {
```

Comment:
Is this change safe? It sounds like it relies on the doctype, which might not be correct.

Reply:
It is. Code before ...

```
        if ($startDate === false) {
            return false;
        }
        return $startDate;
```

This check is useles.